platform/m6/makefile.inc

```diff
@@ -0,0 +1,26 @@
+THUMB_FW=1
+ARMV7A=1
+
+PLATFORMID=12997 # from lsusb
+PLATFORMOS=dryos
+PLATFORMOSVER=59
+DIGIC = 70
+
+MEMBASEADDR=0x8000
+ROMBASEADDR=0xE0000000
+
+override OPT_FI2=
+
+# TODO: check
+MAXRAMADDR = 0x3fffffff
+
+# TODO: check
+NEED_ENCODED_DISKBOOT=19
+KEYSYS=d4k
+FI2FLAGS=W
+
+# next line is for the compatibility check, as comment
+# PLATFORMID @ 0xe1f20270
```

Comment:
not sure about this one

Reply:
Same address applies to your camera too, you'll find the 16-bit platform ID there.